src/screens/Departures/PlaceScreen.tsx

```diff
@@ -72,20 +81,28 @@ export default function PlaceScreen({
       <FlatList
         data={place.quays}
         style={styles.quayChipContainer}
-        horizontal={true}
+        horizontal={!isMissingQuays}
         showsHorizontalScrollIndicator={false}
         keyExtractor={(item) => item.id}
         ListHeaderComponent={
-          <Button
-            onPress={() => {
-              navigation.setParams({selectedQuay: undefined});
-            }}
-            text={t(DeparturesTexts.quayChips.allStops)}
-            color={selectedQuay ? 'secondary_2' : 'secondary_3'}
-            style={[styles.quayChip, {marginLeft: theme.spacings.medium}]}
-            accessibilityHint={t(DeparturesTexts.quayChips.a11yAllStopsHint)}
-            testID="allStopsSelectionButton"
-          ></Button>
+          <>
+            {isMissingQuays ? (
+              <ActivityIndicator size="large"></ActivityIndicator>
```

Comment:
Helst bruk self closing tags, altså her `<ActivityIndicator size="large" />`

Reply:
Sant, finnes det noen ESLint regel for dette? Vet det ikke er første gang jeg glemmer dette, så hadde vært fint å få beskjed av linter.